synapse/federation/federation_server.py

```diff
@@ -454,6 +455,7 @@ async def _on_state_ids_request_compute(self, room_id, event_id):
     async def _on_context_state_request_compute(
         self, room_id: str, event_id: str
     ) -> Dict[str, list]:
+        pdus = []  # type: Iterable[EventBase]
```

Comment:
I think you can add the type two lines below this (the first time `pdus` is declared) instead of allocating a list that will never be used.

Reply:
I didn't do this because it'd look wonky, but alright :+1:

Reply:
Ehhh, i still find it wonky, but i do see a way to make this more pythonic